hack/make-rules/test-cmd.sh

```diff
@@ -598,6 +598,22 @@ runTests() {
   # Post-condition: valid-pod is labelled
   kube::test::get_object_assert 'pod valid-pod' "{{range$labels_field}}{{.}}:{{end}}" 'valid-pod:new-valid-pod:'
 
+  ### Record label change
+  # Pre-condition: valid-pod does not have record annotation
+  kube::test::get_object_assert 'pod valid-pod' "{{range.items}}{{$annotations_field}}:{{end}}" ''
+  # Command
+  kubectl label pods valid-pod record-change=true --record=true "${kube_flags[@]}"
+  # Post-condition: valid-pod has record annotation
+  kube::test::get_object_assert 'pod valid-pod' "{{range$annotations_field}}{{.}}:{{end}}" ".*--record=true.*"
+  
+  ### Do not record label change
+  # Command
+  kubectl label pods valid-pod no-record-change=true "${kube_flags[@]}" --record=false
+  # Post-condition: valid-pod's record annotation still contains command with --record=true
+  kube::test::get_object_assert 'pod valid-pod' "{{range$annotations_field}}{{.}}:{{end}}" ".*--record=true.*"
+    
+    
```

Comment:
Please also add a test that the record is written if the flag is unspecified and the object has a previous record